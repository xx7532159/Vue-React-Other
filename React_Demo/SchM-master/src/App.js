// import '../debug/debuggap';
import 'lib-flexible';
import React from 'react';
import ReactDOM from 'react-dom';
import {Provider} from 'react-redux';
import route from './Router/route'; // 路由配置
import store from './Redux/Store/store';
import './Style/common.scss';
import './Style/coverAntDesign.css';

ReactDOM.render(
    <Provider store={store}>
        {route}
    </Provider>,
    document.getElementById('root')
);
