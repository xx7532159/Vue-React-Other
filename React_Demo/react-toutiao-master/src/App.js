import React from 'react'
import HeaderBar from 'components/HeaderBar'
import FooterBar from 'components/FooterBar'
const mainStyles = {
	width: '100%',
	height: 'calc(100% - 1rem)'
}
class App extends React.Component {
    render () {
        return (
            <div id="container">
                <HeaderBar/>
                <main style={mainStyles}>
                	{this.props.children}
                </main>
                <FooterBar/>
            </div>
        )
    }
}
export default App
