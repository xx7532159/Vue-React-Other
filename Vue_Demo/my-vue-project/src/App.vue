<template>
  <div id="app">

    <keep-alive>
     <router-view v-if="$route.meta.keepAlive"/>
    </keep-alive>
    <router-view v-if="!$route.meta.keepAlive"/>
  </div>
</template>

<script>
import myHeader from "./components/Header/header"
export default {
  name: 'App'
}
</script>

<style>
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>
