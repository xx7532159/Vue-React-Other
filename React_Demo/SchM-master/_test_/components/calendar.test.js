import React from 'react';
// import ReactDOM from 'react-dom';
import Renderer from 'react-test-renderer';
import App from '../wapComponent';
import Calendar from '../../src/Component/calendar/calendar';

const component = Renderer.create(
  <App components={Calendar}/>
);
let tree = component.toJSON();
// console.log(Object.keys(Calendar.childContextTypes));
// const App = WapComponent(Calendar);
it('renders without crashing', () => {
  expect(tree).toMatchSnapshot();
  // const div = document.createElement('div');
  // ReactDOM.render(<App components={Calendar}/>, div);
});

it('swipe', () => {
  // Calendar.onSwipe({direction: 2});
  // const spy = jest.spyOn(Calendar, 'onSwipe');
  // spy({direction: 2});
  // tree.props.onSwipe({direction: 1});
});
