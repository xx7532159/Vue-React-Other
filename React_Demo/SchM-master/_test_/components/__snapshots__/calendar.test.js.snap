exports[`test renders without crashing 1`] = `<components />`;
